flow-data/src/main/java/com/vaadin/flow/data/binder/Binder.java

```diff
@@ -282,6 +282,18 @@ default BindingValidationStatus<TARGET> validate() {
          * @return A boolean value
          */
         boolean isConvertBackToPresentation();
+
+        /**
+         * Checks whether the related Binder instance has marked the field as
+         * having uncommitted changes.
+         *
+         * @throws IllegalStateException
+         *             if {@code binder} is null. Ensure that the binder has
+         *             been declared here.
```

Comment:
I would not mention the Binder field here, as it is an implementation detail.
What about "if the binding is no longer attached to a Binder", as in the exception thrown by `validate(...)`?
```suggestion
         * @throws IllegalStateException
         *             if the binding is no longer attached to a Binder.
```